site/content/en/docs/Tasks/labels_and_annotations.md

```diff
@@ -1,8 +1,46 @@
 ---
 title: "Labels and Annotations"
 linkTitle: "Labels and Annotations"
-weight: 2
+weight: 4
 date: 2017-01-05
 description: >
   Working with Labels and Annotations
----
\ No newline at end of file
+---
+
+{{< alert color="success" title="TL;DR" >}}
+- Set Labels for all Resources declared within a project with `commonLabels`
+- Set Annotations for all Resources declared within a project with `commonAnnotations`
+{{< /alert >}}
+
+## Motivation
+Users may want to define a common set of labels or annotations for all the Resources in a project.
+
```

Comment:
In this sentence, the doc is referring to the users as if they are third party.
So it sort of makes as feel like this doc is for the contributors to read, not the actual users of kustomize users. Same goes for all the other first sentences of **Motivation**.

```suggestion
It's possible to define a common set of labels or annotations for all the Resources in a project.
```

```suggestion
You can define a common set of labels or annotations for all the Resources in a project.
```